NEWS.rst

```diff
@@ -3,6 +3,97 @@ http://twistedmatrix.com/trac/ticket/<number>
 
 .. towncrier release notes start
 
+Twisted 22.4.0.rc1 (2022-04-04)
+===============================
+
+Features
+--------
+
+- twisted.python.failure.Failure tracebacks now capture module information, improving compatibility with the Raven Sentry client. (#7796)
+- twisted.python.failure.Failure objects are now compatible with dis.distb, improving compatibility with post-mortem debuggers. (#9599)
+
+
+Bugfixes
+--------
+
+- Correct type annotations for IReactorSSL.listenSSL. (#10274)
+- twisted.internet.test.test_glibbase.GlibReactorBaseTests now passes. (#10317)
+
+
+Conch
+-----
+
+Features
+~~~~~~~~
+
+- twisted.conch.ssh now supports using RSA keys with SHA-2 signatures (RFC 8332) when acting as a server.  The rsa-sha2-512 and rsa-sha2-256 public key signature algorithms are automatically preferred over ssh-rsa if the client advertises support for them; the actual public keys do not need to change. (#9765)
+- twisted.conch.ssh now has an alternative Ed25519 implementation using PyNaCl, in order to support platforms that lack OpenSSL >= 1.1.1b.  The new "conch_nacl" extra has the necessary dependency. (#10208)
+
+
+Bugfixes
+~~~~~~~~
+
+- twisted.conch.ssh.session.SSHSession now cleans up both the client transport and the ISession adapter if both are set.  Previously, a subsystem's connectionLost method was not called if a environment variable passing request was also sent on the same channel. (#10308)
+-  (#10313)
```

Comment:
[This ticket](https://twistedmatrix.com/trac/ticket/10313) is a fix for a regression caused by the previous fix, so it can be omitted from the release notes.

Reply:
It looks like it's still here.  This should be moved to the misc section, I think.